Generals/Code/GameEngine/CMakeLists.txt

```diff
@@ -457,60 +457,11 @@ set(GAMEENGINE_SRC
     Include/GameLogic/WeaponSetFlags.h
     Include/GameLogic/WeaponSetType.h
     Include/GameLogic/WeaponStatus.h
-    Include/GameNetwork/Connection.h
-    Include/GameNetwork/ConnectionManager.h
-    Include/GameNetwork/DisconnectManager.h
-    Include/GameNetwork/DownloadManager.h
-    Include/GameNetwork/FileTransfer.h
-    Include/GameNetwork/FirewallHelper.h
-    Include/GameNetwork/FrameData.h
-    Include/GameNetwork/FrameDataManager.h
-    Include/GameNetwork/FrameMetrics.h
-    Include/GameNetwork/GameInfo.h
-    Include/GameNetwork/GameMessageParser.h
+    #Include/GameNetwork/DisconnectManager.h
+    #Include/GameNetwork/GameInfo.h
     Include/GameNetwork/GameSpy.h
-    Include/GameNetwork/GameSpy/BuddyDefs.h
-    Include/GameNetwork/GameSpy/BuddyThread.h
-    Include/GameNetwork/GameSpy/GameResultsThread.h
-    Include/GameNetwork/GameSpy/GSConfig.h
-    Include/GameNetwork/GameSpy/LadderDefs.h
-    Include/GameNetwork/GameSpy/LobbyUtils.h
-    Include/GameNetwork/GameSpy/MainMenuUtils.h
-    Include/GameNetwork/GameSpy/PeerDefs.h
-    Include/GameNetwork/GameSpy/PeerDefsImplementation.h
-    Include/GameNetwork/GameSpy/PeerThread.h
-    Include/GameNetwork/GameSpy/PersistentStorageDefs.h
-    Include/GameNetwork/GameSpy/PersistentStorageThread.h
-    Include/GameNetwork/GameSpy/PingThread.h
-    Include/GameNetwork/GameSpy/StagingRoomGameInfo.h
-    Include/GameNetwork/GameSpy/ThreadUtils.h
-    Include/GameNetwork/GameSpyChat.h
-    Include/GameNetwork/GameSpyGameInfo.h
-    Include/GameNetwork/GameSpyGP.h
-    Include/GameNetwork/GameSpyOverlay.h
     Include/GameNetwork/GameSpyPersistentStorage.h
-    Include/GameNetwork/GameSpyThread.h
     Include/GameNetwork/GUIUtil.h
```

Comment:
Why are some of the files not moved here? If I am not mistaken all except GUIUtil.cpp should be matching.